src/EditorFeatures/Core.Wpf/InlineErrors/InlineErrorsLocations.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.CodeAnalysis.Editor.InlineErrors
+{
+    internal enum InlineErrorsLocations
+    {
+        HookedToCode,
+        HookedToWindow,
+
+        Default = HookedToCode,
```

Comment:
why?

Reply:
removed, was matching the pattern of a different struct